### ISSUE TITLE
Alignment problem for collision shapes when using isometric maps

### ISSUE THREAD

**Issue by msoledade** [vnen/godot-tiled-importer#66]
When using Tiled for isometric maps and adding a collision shape, it gets transposed to orthogonal on Godot.

![tiled](https://user-images.githubusercontent.com/1319498/35774595-5dac2d70-095b-11e8-8855-473f40202f99.png)

![godot](https://user-images.githubusercontent.com/1319498/35774602-6475dcbe-095b-11e8-96d7-8003a69fc184.png)

Using on Windows 10, with Tiled 1.0.3 and Godot 3.0.

**Comment by vnen** [comment]
Yeah, Tiled applies the isometric transform to everything, effectively changing the coordinate system. I need to make some tests, because I believe that applying such transform to collision shapes won't work well with Godot physics. I'll have to investigate a proper solution.

**Comment by slevy85** [comment]
Hi, thank you for this great plugin,
I have the same problem, I am not sure how should I deal with it.
Do you know how to make it work ?

**Comment by vnen** [comment]
There's not any easy to solve this. The map converter would need to apply the transform to the shapes, but I'm not sure how Godot physics will behave. Maybe the plugin needs to make a polygon instead.

The problem is that I don't have to work on this, so any help is appreciated.

**Comment by Dinopaterno22** [comment]
I just discovered this issue by myself and made sure to check if the issue had been raised. I'll take a look myself and suggest a modification if I find something in my spare time :)

**Comment by elvisish** [comment]
This is definitely a problem, here's how my map looks in TilEd:
![image](https://user-images.githubusercontent.com/16231628/120044871-a2499480-c006-11eb-9868-71c8b1ff83d0.png)
and here it is in GD:
![image](https://user-images.githubusercontent.com/16231628/120044889-a83f7580-c006-11eb-8996-f5e8b14bb3a9.png)
any ideas how to quick fix this?
